changelog/fix_bug_in_stylelambdacall.md

```diff
@@ -0,0 +1 @@
+* [#9225](https://github.com/rubocop-hq/rubocop/issues/9225): Fix bug in Style/LambdaCall. ([@sswander][])
```

Comment:
Better to add a more specific message here, like the PR title.

Reply:
Thanks @fatkodima ! Updated.